packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -149,6 +149,35 @@ const buttonStyles: ComponentSlotStylesPrepared<ButtonProps, ButtonVariables> =
           },
         }),
 
+      ...(p.inverted && {
+        outline: 0,
```

Comment:
Do we ever want an outline on button? Should we move this to all buttons - ie outside of `p.inverted`?

Reply:
that's an artifact - removed. It's also on the 'rectangle'.